app/assets/stylesheets/views/_taxons.scss

```diff
@@ -35,6 +35,59 @@
   margin-top: 0;
 }
 
+.taxon-page__featured-item {
+  @extend %contain-floats;
+  margin-top: $gutter-two-thirds;
+
+  &--single {
+    min-width: 330px;
+  }
+
+  .taxon-page__featured-image {
+    @include box-sizing(border-box);
+    width: (1 / 3) * 100%;
+    float: left;
+    margin-bottom: $gutter-one-third;
+    padding-right: $gutter-half;
+
+    @include media(tablet) {
+      float: none;
+      margin-bottom: $gutter-one-third;
+      padding-right: 0;
+      width: 100%;
+    }
+  }
+
+  .taxon-page__featured-text {
+    display: inline-block;
```

Comment:
Interestingly you can remove all of the CSS for this element and it still behaves (almost) correctly, because the image next to it is floated left. Unfortunately that doesn't work if at any point the text is taller than the image, because it wraps onto the next line. I'd advise for consistency to also float this element, even though you'd have to add another line to turn off float above tablet.